src/types/models/Ctx.d.ts

```diff
@@ -42,6 +42,13 @@ type Profile = {
     settings: Settings,
 };
 
+type Notifications = {
+    uid: string,
+    created: string,
+    items: Record<string, Video[]>,
```

Comment:
This is no more a Video. Instead it's a NotificationItem with videoId, metaId & videoReleased.
- [ ] Update